testing/endpoint.go

```diff
@@ -117,7 +123,12 @@ func (endpoint *Endpoint) CreateClient() (err error) {
 		if err != nil {
 			return err
 		}
-		clientState = wasmtypes.NewClientState(wasmClientState, endpoint.Chain.Coordinator.CodeHash, height)
+
+		wasmTendermintConfig, ok := endpoint.ClientConfig.(*WasmTendermintConfig)
```

Comment:
I think this fails before we reach this code. On lines 114-115:

```go
tmConfig, ok := endpoint.ClientConfig.(*TendermintConfig)
require.True(endpoint.Chain.TB, ok)
```

maybe we move this to the top of the swtich case?

Reply:
Wasn't sure on ^ point too but yeah, it should fail [as per the spec](https://go.dev/ref/spec#Type_assertions):

> More precisely, if T is not an interface type, x.(T) asserts that the dynamic type of x is [identical](https://go.dev/ref/spec#Type_identity) to the type T